linera-execution/src/lib.rs

```diff
@@ -236,6 +236,8 @@ pub enum ExecutionError {
     MaximumServiceOracleExecutionTimeExceeded,
     #[error("Serialized size of the executed block exceeds limit")]
     ExecutedBlockTooLarge,
+    #[error("HTTP response exceeds the size limit of {limit} bytes, having at least {size} bytes")]
+    HttpResponseSizeLimitExceeded { limit: u64, size: u64 },
```

Comment:
Should we make this a limit for all types of oracle responses?

Reply:
Oh right, I'll open an issue to limit the oracle response size as well.

Reply:
Issue #3563 created.